src/features/virtualized-table/VirtualizedTable.js

```diff
@@ -10,17 +10,17 @@ import './VirtualizedTable.scss';
 
 type SortHandler = SortParams => void;
 
-type Props = {
+export type VirtualizedTableProps = {
     children: Node | (any => Node),
     className?: string,
     height?: number,
-    intl: any,
+    intl?: any,
```

Comment:
I think the type is `InjectIntlProvidedProps`, no?

Reply:
Ah yes, missed this. Will update